cli/op_fetch_asset.rs

```diff
@@ -85,6 +83,7 @@ pub fn get_asset(name: &str) -> Option<&'static str> {
 
 /// Warning: Returns a non-JSON op dispatcher. Must be manually attached to
 /// JsRuntime.
+#[allow(unused)]
```

Comment:
Is this actually unused?

Reply:
Yeah, I meant to comment on that, it is required for the time being, as it is used by the snapshot builder, but not by the `main.rs` anymore, so it errors as unused if removed.  A subsequent refactor of this whole thing needs to be done to clean up the building of the snapshot, and then having some assets on demand.